templates/homev2_wifi.tpl

```diff
@@ -28,6 +28,8 @@
 #include <LTR329.h>
 #include <ArduinoBearSSL.h>
 #include <Adafruit_DPS310.h> // http://librarymanager/All#Adafruit_DPS310
+#include <sps30.h> // http://librarymanager/All#
```

Comment:
If the library is not available in the arduino librarymanager, you can remove the comment

Reply:
9d6221a13a03013c85b6606629bb6fd5923c56eb